charts/stable/valheim/questions.yaml

```diff
@@ -9,205 +9,183 @@ questions:
 # Include{replica1}
 # Include{controllerExpertExtraArgs}
   - variable: secretEnv
-    group: "App Configuration"
-    label: "Image Secrets"
+    group: App Configuration
+    label: Image Secrets
     schema:
       additional_attrs: true
       type: dict
       attrs:
         - variable: SUPERVISOR_HTTP_USER
-          label: "SUPERVISOR_HTTP_USER"
+          label: Supervisor User
           schema:
             type: string
-            default: "admin"
+            default: ''
             required: true
         - variable: SUPERVISOR_HTTP_PASS
-          label: "SUPERVISOR_HTTP_PASS"
+          label: Supervisor PASS
           schema:
             type: string
             required: true
             private: true
-            default: "REPLACETHIS"
+            default: ''
         - variable: SERVER_PASS
-          label: "SERVER_PASS"
+          label: Server Pass
           schema:
             type: string
             required: true
             private: true
-            default: "REPLACETHIS"
+            default: ''
```

Comment:
```suggestion
            default: ""
```

Reply:
Is there any kind of style guide or documentation of conventions for this, and similar things?

Reply:
Nop, but YAML mostly uses double quotes and single quotes when it's necessary.
Nothing set in stone.

Reply:
I use single quotes when string is empty so i don't leave it without any quotes and it fails the cli.